internal/DateComponentPicker/DateComponentPicker.module.less

```diff
@@ -30,5 +29,19 @@
 			opacity: @agate-datetimepicker-divider-opacity;
 			background-color: @agate-datetimepicker-divider-bg-color;
 		}
+
+		&:nth-child(n+2)::before {
+			content: "";
+		}
+
+		:global(.enact-locale-right-to-left) & {
+			&:nth-child(n+2)::before {
```

Comment:
I think that `n + 3`  would be better because
if not, the 2nd picker will be applied with the following 3 kinds of styles.
If using `n + 3`, then we will be applied with the first and the third styles in the following one.

```
.enact-locale-right-to-left .DateComponentPicker_dateComponentPicker__3e2Ns.gallium:nth-child(-n + 2)::before {
    content: "";
}
.enact-locale-right-to-left .DateComponentPicker_dateComponentPicker__3e2Ns.gallium:nth-child(n + 2)::before {
    content: none;
}
.DateComponentPicker_dateComponentPicker__3e2Ns.gallium:nth-child(n + 2)::before {
    content: "";
}
```